tests/test_configmanager.py

```diff
@@ -133,24 +136,23 @@ def setUp(self):
 
     def test_parse(self):
         self.configmanager.parse()
-        self.assertIn('eth0', self.configmanager.ethernets)
-        self.assertIn('bond6', self.configmanager.bonds)
+        state = self.configmanager.np_state
+        assert state
+        self.assertIn('eth0', state.ethernets)
+        self.assertIn('bond6', state.bonds)
```

Comment:
IIUC `self.configmanager.ethernets/bonds` should work as well, going through config_manager's `__getattr__` method to fetch the data from `np_state`, right?

Reply:
That's right. We don't test this feature explicitly, but it get exercised a bit in the sriov codepaths, I believe.